packages/portal-proto/src/features/facets/ToggleFacet.unit.test.tsx

```diff
@@ -30,9 +30,7 @@ describe("<ToggleFacet />", () => {
       />,
     );
 
-    const ctrl = getByRole("checkbox", {
-      name: "toggle facet value",
-    });
+    const ctrl = getByTestId("toggle-facet-value");
```

Comment:
Was this selector not working anymore? General convention for react testing library is to prefer selecting by role over test id: https://testing-library.com/docs/queries/about#priority

Reply:
it was not working because the aria label was remove in favor of a label with the "for" which i am updating to the more correct htmlFor